### ISSUE TITLE
Setting env and tagName together makes rule fail silently

### ISSUE THREAD

**Issue by garcianavalon** [apollographql/eslint-plugin-graphql#256]
Specifying both `env` and `tagName`  is causing rules to silently fail internally with out any errors:

```js
// this works 
'graphql/template-strings': [
  'error', {
    // I'm using .graphqlconfig to import the schema
    env: 'literal',
}],
```
```js
// this fails silently ->  eslint will say the file is valid, no error shown nor thrown
'graphql/template-strings': [
  'error', {
    // I'm using .graphqlconfig to import the schema
    env: 'literal',
    tagName: 'gql'  // <- don't do this
}],
```

I think this should not happen. I don`t know whats wrong with both using `env` and `tagName` but it should not fail silently and this should be considered a bug

_Originally posted by @garcianavalon in https://github.com/apollographql/eslint-plugin-graphql/issues/252#issuecomment-577167742_
